### DISCUSSION TITLE
Sizing magnitudes of weights in prior information equations of PEST_HP

### DISCUSSION THREAD

**Discussion by sjepsen395** [usgs/pestpp#305] [Q&A] [answered]
Hi All,

I'm a little lost on how to set reasonable magnitudes of weights in prior information equations of PEST_HP. When I set all weights to 1, my regularization objective is 8 orders of magnitude lower than my measurement objective (# observations >> # regularization groups). Then, when I set all weights to 1000, the regularization objective increased to around 0.1% of the measurement objective. Surprisingly, the final parameter values are different between the weight = 1 and 1000 cases (I expected no difference because of how PEST_HP automatically calculates weights to attain a target objective). I thought my changes to weight magnitudes would be negated by automatic weight calculations in PEST_HP, which left me perplexed. Is there by chance a rule of thumb on what the final regularization objective should be relative to the final measurement objective? Maybe that could guide how weights should sized. IES has an option to set the regularization objective to a user defined fraction of the measurement objective, something like 10% if I remember correctly. Does that mean that it would make sense to further increase my weights from 1000 until my final (regularization objective)/(measurement objective) is about 0.1? I'm not even sure if I could get a solution with those settings.

Thanks,
Steve

**Comment by jtwhite79** [comment]
I think in general its not recommended to set the weights on PI equations manually...but if you want to, you'll need to keep monkeying with weights between obs and PI equations until the phi value is the proportions you like.  Its going to be problem specific...

**Reply by sjepsen395** [reply]
jtwhite79,

I'm finding in PEST_HP that I need to manually set my prior information (PI) weights to control the final regularization objective in relation to the measurement objective. For example, on my initial test run, I left all PI weights to their 1.0 defaults. The resulting final regularization was only 5.2 compared to a measurement objective of 3.2e+8. That seemed too "gentle" of regularization. Increasing all PI weights to 247 resulted in the final regularization objective increasing to 294,856. Maybe getting a little closer. I originally thought that PEST_HP automatically handles weight scaling, but now I'm not so sure. Perhaps I'm just not looking at outputs in the right way. Or maybe I'm missing a parameter adjustment. I'm using a IREGADJ setting of 1.

Can I please ask you this? If I wanted to capture in PEST_HP the below text from pg. 145 of PEST++ User's Guide v5.2.10, would I adjust my PI weights so that the final regularization objective is 25% of the final measurement objective? Is that the right idea?:

"Where regularization is applied, determination of a suitable value for ies_reg_factor() can be difficult. Its value depends on the nature of the inverse problem, on the number of parameters comprising a realization, and on the desired level of model-to-measurement misfit. Despite its dependence on all of these variables, a value in the vicinity of 0.25 often works well."

Thanks,
Steve

**Reply by jtwhite79** [reply, marked as answer]
if you are using pest in "regularization" mode, it dynamically adjusts the contribution of the PI equations to the composite phi each iteration until the phimlim and/or fracphim values are satisfied so I dont think you need to worry about it...check out this for more deets:

[https://pubs.usgs.gov/sir/2010/5169/pdf/GWPEST_sir2010-5169.pdf](https://pubs.usgs.gov/sir/2010/5169/pdf/GWPEST_sir2010-5169.pdf)

but yeah if you wanted 25% of composite phi to be from PI eqs, then just keep pumping up the weights on PI eqs until you get that balance...

regularization in IES is a different beast since what you are regularizing is each realization to stay as much like its prior values...

**Reply by sjepsen395** [reply]
jtwhite79,

Thanks a lot for the information. I'll have a look at that paper.

Steve
